business_calendar_test.go

```diff
@@ -337,3 +337,35 @@ func Test_businessCalendar_ConsistencyChecks(t *testing.T) {
 		}
 	})
 }
+
+func benchmarkShiftBusinessDays(b *testing.B, shift int, date date.Date) {
```

Comment:
Factorize everything in benchmark cases without helper (they work exactly as test cases):
```go
calendar := newBusinessCalendar()
for _, bc := range []struct {
   name string
   date date.Date
   shift int
}{
   // etc.
}
```